CANTroller2/src/globals.h

```diff
@@ -229,9 +230,9 @@ temp_status temp_state = CONVERT;
 uint32_t temp_times_us[2] = { 2000000, 10000 };  // Peef delay was 10000 (10ms)
 uint32_t temp_timeout_us = 2000000;
 Timer tempTimer (temp_timeout_us);
-DeviceAddress temp_temp_addr;
-DeviceAddress temp_addrs[6];  // Hard code to the actual sensor addresses for the corresponding sense location on the car
-DallasSensor tempsensebus (&onewire);
+
+TemperatureSensorManager temperature_sensor_manager;
```

Comment:
you should move all the globals that are only used internally to inside the class (like temp_status, temp_state, tempTimer, etc). ideally we would have ALL the temperature state handled in one place, so instead of a global temps_f array we'd have accessors from the temperature manager (e.g. temp_manager.get_ambient_min() instead of temps_f[AMBIENT][MIN]), but we can do that kind of clean up later. We don't want random unused globals sitting around tho.

Reply:
Yeah this min max stuff is still a todo.